requirements/base.txt

```diff
@@ -1,5 +1,5 @@
-arrow>=0.12,<0.13
+arrow>=0.14,<0.15
```

Comment:
Can we leave the lower end at 0.12 or does this change not work on < 0.14?

Reply:
I haven't specifically tried it with .12, I was following the pattern elsewhere in the file where it seemed we tried to pin to approximately 1 minor version. I'll give it a shot and report back (here, and below)